dask/array/core.py

```diff
@@ -1307,6 +1307,10 @@ def ravel(self):
 
     flatten = ravel
 
+    @wraps(np.choose)
+    def choose(self, choices):
+        return elemwise(variadic_choose, self, *choices)
```

Comment:
That's a good point! It probably makes more sense to just implement `choose` on a dask Array once. I'll change the PR to have the `choose` function simply call the `choose` method on an input dask array.